cookbook/specs/spec_add_manta.json

```diff
@@ -0,0 +1,115 @@
+{
+    "proposal": {
+      "title": "Add Specs: Manta",
+      "description": "Adding new specification support for relaying data on Manta",
+      "specs": [
+        {
+          "index": "MANTA",
+          "name": "manta mainnet",
+          "enabled": true,
+          "imports": ["ETH1"],
+          "reliability_threshold": 268435455,
+          "data_reliability_enabled": true,
+          "block_distance_for_finalized_data": 1,
+          "blocks_in_finalization_proof": 3,
+          "average_block_time": 10000,
+          "allowed_block_lag_for_qos_sync": 5,
+          "shares": 1,
+          "min_stake_provider": {
+            "denom": "ulava",
+            "amount": "47500000000"
+          },
+          "api_collections": [
+            {
+              "enabled": true,
+              "collection_data": {
+                "api_interface": "jsonrpc",
+                "internal_path": "",
+                "type": "POST",
+                "add_on": ""
+              },
+              "apis": [],
+              "headers": [],
+              "inheritance_apis": [],
+              "parse_directives": [],
+              "verifications": [
+                {
+                  "name": "chain-id",
+                  "values": [
+                    {
+                      "expected_value": "0xa9"
+                    }
+                  ]
+                },
+                {
+                  "name": "pruning",
+                  "values": [
+                    {
+                      "latest_distance": 0
```

Comment:
0 here means any pruning value will work, this is not working well with the archive value, this works together with the extension archive block rule, rule of thumb is 24h of blocks

Reply:
just to clarify, @omerlavanet means that the value that you need to put is:
<manta_average_block_time_in_seconds> * <number_of_seconds_in_24H> = number of blocks in 24H

Reply:
Also changed this to 8640 block